src/editor.py

```diff
@@ -263,12 +264,12 @@ def input_colorPicker(self, val):
 			if val.name == "KEY_ESCAPE" or val.name == "KEY_ENTER":
 				self.colorPickerFieldSelected = False
 		else:
-			if val.name == "KEY_RIGHT" and self.colorPickerSelectedCol < 3:
-				self.colorPickerColor[self.colorPickerSelectedCol] += 1
+			if val.name in ("KEY_RIGHT", "KEY_SRIGHT") and self.colorPickerSelectedCol < 3:
+				self.colorPickerColor[self.colorPickerSelectedCol] += {'KEY_RIGHT': 1, 'KEY_SRIGHT': 10}[val.name]
```

Comment:
oooh, didn't know that `{}[]` was a thing!

Reply:
It's a pattern of "keying" into a dictionary, instead of using if statements.
```
if a == "1":
    result = "2"
elif a == "2":
    result = "3"
else:
   result = "4"
```

can be changed to,
```
result = {"1": 2", "2": "3"}.get(a, "4")
```